src/pages/Modal.jsx

```diff
@@ -3,8 +3,10 @@ import { motion } from "framer-motion";
 import { ThemeContext } from "../Context&functions/ThemeContext";
 const ProductDetails = lazy(() => import("./components/productDetails"));
 import { AnimatePresence } from "framer-motion";
+
 const Modal = ({ isOpen, setIsOpen, product }) => {
   const { theme } = useContext(ThemeContext);
+
   const bgColor =
     theme === "dark" ? "bg-gray-900 text-white" : "bg-gray-100 text-white";
```

Comment:
Don't you make these guys constants, since the are being used more than once in different places, OR create a global function to get the theme.